cross/tvheadend/Makefile

```diff
@@ -1,11 +1,11 @@
 PKG_NAME = tvheadend
 PKG_VERS = 4.3
 PKG_EXT = tar.gz
-PKG_DOWNLOAD_METHOD = git
 PKG_GIT_HASH = 9ed76c0a176b055a57b6e8bd2e0b6e29409269a9
-PKG_DIST_SITE = https://github.com/tvheadend/tvheadend.git
+PKG_DIST_NAME = $(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/tvheadend/tvheadend/archive
 PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
-PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
```

Comment:
Not sure I follow, will this download the old 4.3 version or the version matching the git hash?
As I never saw that option of downloading a tar.gz from a git hash though the archive URL path.

Reply:
It will download the state the repository is in at particular hash.

If you go to https://github.com/tvheadend/tvheadend/tree/9ed76c0a176b055a57b6e8bd2e0b6e29409269a9 and hit the green `code button` and hover over `download zip` you should see the URL I used. (I just changed the file extension)

Reply:
thnx, nice to know.
I guess we should change other package in using that same method instead of git.